### PR TITLE
Update all translations

### PR DESCRIPTION
**Pull request by tchx84**

### CODE REVIEW

**Comment by RacerBG** [comment]
Is it machine translation?

Anyway the only thing which should be changed for the Bulgarian translation is the following string:

"Последно избран идентификатор на приложение" should become "Последно избран идентификатор на приложението".

The other strings looks fine to my surprise. :smile:

**Comment by tchx84** [comment]
> Is it machine translation?

Yes, I usually do this and then ask people to improve it :sweat_smile: 

> 
> Anyway the only thing which should be changed for the Bulgarian translation is the following string:
> 
> "Последно избран идентификатор на приложение" should become "Последно избран идентификатор на приложението".

thanks! Will change.

> 
> The other strings looks fine to my surprise. 😄

t

**Comment by tchx84** [comment]
Hi! 

I auto-updated all translations with 3 new strings, in case you can inspect it ! Thanks :raised_hands: 

@AsciiWolf @ovari @Vistaus @eson57 @cho2 @MiloCasagrande @BigmenPixel0 @juxuanu @kbdharun @EricZhang456 @RacerBG @yarons @wsxy162 @daPhipz @Garbulix @stenosis @NikolaiES @Vovkiv @mbousq @sabriunal @Garbulix @uslmz

**Comment by ghost** [comment]
I checked Turkish and seems adequate.

File: po/ta.po [hunk 1]
```diff
@@ -48,7 +48,7 @@ msgstr "Martin Abente Lahaye"
 
 #: data/com.github.tchx84.Flatseal.gschema.xml:15
 msgid "Application ID that was last selected"
-msgstr ""
+msgstr "கடைசியாக தேர்ந்தெடுக்கப்பட்ட விண்ணப்ப ஐடி"
```
Comment (kbdharun) [inline po/ta.po:51]:
```suggestion
msgstr "கடைசியாக தேர்ந்தெடுக்கப்பட்ட பயன்பாடு ஐடி"
```

Thanks for the modification, The other strings LGTM. In the above string, விண்ணப்பம் means envelope which also means applications (but business ones) in this case the widely used term for application/computer program is பயன்பாடு. Will open a PR to update the same.

File: po/ca.po [hunk 2]
```diff
@@ -579,11 +579,11 @@ msgstr "_Desfés"
 
 #: src/widgets/window.js:108
 msgid "Cannot load overrides due to incorrect contents"
-msgstr ""
+msgstr "No es poden carregar els canvis a causa d'un contingut incorrecte"
```
Comment (juxuanu) [inline po/ca.po:582]:
```suggestion
msgstr "No es poden carregar les redefinicions a causa d'incorreccions en el seu contingut"
```

File: po/ca.po [hunk 3]
```diff
@@ -579,11 +579,11 @@ msgstr "_Desfés"
...
 
 #: src/widgets/window.js:115
 msgid "Refreshed due to changes in Flatpak installations"
-msgstr ""
+msgstr "Actualitzat a causa dels canvis a les instal·lacions de Flatpak"
```
Comment (juxuanu) [inline po/ca.po:586]:
```suggestion
msgstr "Actualitzat a causa de canvis a la instal·lació del Flatpak"
```

**Comment by AsciiWolf** [comment]
The Czech translation looks good!

**Comment by daPhipz** [comment]
The German (de) translation looks good!

**Comment by juxuanu** [comment]
@tchx84 will you add the suggested changes?
Next time please wait before merging so we can give feedback

**Comment by AsciiWolf** [comment]
@juxuanu Directly editing the po files by hand is not ideal.

**Comment by tchx84** [comment]
> Hi!
> 
> I auto-updated all translations with 3 new strings, in case you can inspect it ! Thanks 🙌
> 
> @AsciiWolf @ovari @Vistaus @eson57 @cho2 @MiloCasagrande @BigmenPixel0 @juxuanu @kbdharun @EricZhang456 @RacerBG @yarons @wsxy162 @daPhipz @Garbulix @stenosis @NikolaiES @Vovkiv @mbousq @sabriunal @Garbulix @uslmz

Apologies, I wasn't clear. I will appreciate if you can send a PR with the improvements :pray:

**Comment by MiloCasagrande** [comment]
@tchx84 Italian translation PR is #574

**Comment by eson57** [comment]
Swedish is OK.

eson57

torsdag den 14 september 2023 skrev AsciiWolf:
> @juxuanu Directly editing the po files by hand is not ideal.
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/tchx84/Flatseal/pull/568#issuecomment-1719258661
> You are receiving this because you were mentioned.
> 
> Message ID: ***@***.***

-- 
Sent from SailfishX

**Comment by Garbulix** [comment]
Polish also looks good.

**Comment by EricZhang456** [comment]
Simplified Chinese looks alright

File: po/he.po [hunk 4]
```diff
@@ -46,7 +46,7 @@ msgstr ""
 
 #: data/com.github.tchx84.Flatseal.gschema.xml:15
 msgid "Application ID that was last selected"
-msgstr ""
+msgstr "מזהה אפליקציה שנבחר לאחרונה"
```
Comment (yarons) [inline po/he.po:49]: מזהה היישומון האחרון שנבחר

File: po/he.po [hunk 5]
```diff
@@ -575,11 +575,11 @@ msgstr "_ביטול"
 
 #: src/widgets/window.js:108
 msgid "Cannot load overrides due to incorrect contents"
-msgstr ""
+msgstr "לא ניתן לטעון שינויים עקב תוכן שגוי"
 
 #: src/widgets/window.js:115
 msgid "Refreshed due to changes in Flatpak installations"
-msgstr ""
+msgstr "רענן עקב שינויים בהתקנות Flatpak"
```
Comment (yarons) [inline po/he.po:582]: רוענן עקב שינויים בהתקנות Flatpak

**Pull request merged**